### DISCUSSION TITLE
How are syntax files (*.sdef) made?

### DISCUSSION THREAD

**Discussion by rchastain** [mse-org/mseide-msegui#70] [General]
Do you know how are syntax files (*.sdef) made?

Which rules do they use? Is it a creation by Martin or it used also by other softwares?

I would be interested in a Markdown syntax file.

**Comment by fredvs** [comment]
Hello Roland.

I did play with custom syntax files following the notes from the original code of the files.
It is clear and easy but I dont know the rules it uses.

**Reply by rchastain** [reply]
Hello Fred. OK, I will see if I manage to understand it.
